packages/common/container-definitions/src/runtime.ts

```diff
@@ -198,6 +198,11 @@ export interface IContainerContext {
 	 * @privateremarks Tracking in AB#5714
 	 */
 	readonly id: string;
+
+	/**
+	 * This snapshot contains other artifacts too like blobContents, ops etc.
+	 */
+	readonly snapshot?: ISnapshot;
```

Comment:
Also, should baseSnapshot be marked as deprecated? It's nuanced message, and maybe we should do it a month from now.

Reply:
Yes, let's not deprecate it now. If in future, we see that it is totally redundant, then we would deprecate that.